extensions/ql-vscode/src/cli.ts

```diff
@@ -608,6 +608,21 @@ export class CodeQLCliServer implements Disposable {
  
     return await this.runCodeQlCliCommand(['database', 'unbundle'], subcommandArgs, `Extracting ${archivePath} to directory ${target}`);
   }
+  
+  /**
+   * Uses a .qhelp file to generate Query Help documentation in a specified format.
+   * @param pathToQlhelp The path to the .qhelp file
```

Comment:
Here and in other places: `qlhelp` should be `qhelp`. (For consistency, we try to use "query help" as the full name and "qhelp" as the abbreviation!)